AdyenDropIn/Presentation/NavigationProtocol.swift

```diff
@@ -0,0 +1,13 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+internal protocol NavigationProtocol: PresentationDelegate {
```

Comment:
not sure about the name NavigationProtocol? seems too generic, what's wrong with presentationDelegate? or even better AdyenPresentationDelegate? it is a delegate after all, having the delegate in its name make sense to me

Reply:
It is internal :)
Check updated version